src/sdk/bucketspace_fs.js

```diff
@@ -344,9 +345,9 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
 
     new_bucket_defaults(account, { name, tag, lock_enabled, force_md5_etag }, create_uls, bucket_storage_path) {
         return {
+            _id: mongo_utils.mongoObjectId(),
             name,
             tag: js_utils.default_value(tag, undefined),
-            owner_account: account._id,
```

Comment:
sorry for telling you to remove it, we will soon move to refer owner account / principal of bucket policy by id, so we will need that. please add it again here and in the schema.

Reply:
@romayalon no issues, I will add the `owner_account`